srcpkgs/backblaze-b2/template

```diff
@@ -1,11 +1,19 @@
 # Template file for 'backblaze-b2'
 pkgname=backblaze-b2
 version=4.2.0
-revision=1
+revision=2
 build_style=python3-pep517
 hostmakedepends="python3 python3-build python3-packaging python3-pdm-backend python3-pip python3-setuptools_scm python3-wheel"
-depends="python3-Arrow python3-b2sdk python3-docutils
- python3-phx-class-registry python3-rst2ansi python3-tabulate"
+depends="python3-annotated-types \
+ python3-argcomplete \
+ python3-Arrow \
+ python3-b2sdk \
+ python3-docutils
+ python3-phx-class-registry \
+ python3-rst2ansi \
+ python3-tabulate \
+ python3-tqdm \
+ python3-platformdirs"
```

Comment:
don't put these one-per-line, and don't use `\`s for continuations

also, some of these do not belong on this package, as they are really dependencies of dependencies.

the missing dependencies, as found by #53326:
```
=> backblaze-b2-4.2.0_1: running pre-pkg hook: 06-verify-python-deps ...
   PYTHON: py3:argcomplete <-> python3-argcomplete NOT IN depends PLEASE FIX!
   PYTHON: py3:arrow <-> python3-Arrow
   PYTHON: py3:b2sdk <-> python3-b2sdk
   PYTHON: py3:docutils <-> python3-docutils
   PYTHON: py3:phx-class-registry <-> python3-phx-class-registry
   PYTHON: py3:rst2ansi <-> python3-rst2ansi
   PYTHON: py3:tabulate <-> python3-tabulate
   PYTHON: py3:tqdm <-> python3-tqdm NOT IN depends PLEASE FIX!
   PYTHON: py3:platformdirs <-> python3-platformdirs NOT IN depends PLEASE FIX!
```
```
=> python3-b2sdk-2.6.0_1: running pre-pkg hook: 06-verify-python-deps ...
   PYTHON: py3:annotated-types <-> python3-annotated-types NOT IN depends PLEASE FIX!
   PYTHON: py3:logfury <-> python3-logfury
   PYTHON: py3:requests <-> python3-requests
```